test/ut/retiarii/test_highlevel_apis.py

```diff
@@ -568,6 +568,45 @@ def forward(self, x):
                 model = mutator.bind_sampler(sampler).apply(model)
             self.assertTrue(self._get_converted_pytorch_model(model)(torch.randn(1, 16)).size() == torch.Size([1, 64]))
 
+    def test_cell_predecessors(self):
+        from typing import List, Tuple
+
+        class Preprocessor(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = nn.Linear(3, 16)
+
+            def forward(self, x: List[torch.Tensor]) -> List[torch.Tensor]:
+                return [self.linear(x[0]), x[1]]
+
+        class Postprocessor(nn.Module):
+            def forward(self, this: torch.Tensor, prev: List[torch.Tensor]) -> Tuple[torch.Tensor, torch.Tensor]:
+                return prev[-1], this
+
+        @self.get_serializer()
```

Comment:
use model_wrapper here?

Reply:
ok

Reply:
why replace @self.get_serializer() by model_wrapper?

Reply:
In the old past, putting `@model_wrapper` on models used by graph engine makes the engine crash, while python engine needs `@model_wrapper`. As you see, `self.get_serializer()` have different implementations between super-class and sub-class.

But this bug has been fixed. We can always apply `@model_wrapper` now.